site/_data/banner.yml

```diff
@@ -1,6 +1,6 @@
 type: info
-text: Check out what's new for the web platform at Google IO!
+text: The PWA community is coming together for PWASummit22. Have a great story about developing a web app?
```

Comment:
In https://github.com/GoogleChrome/web.dev/pull/8246/files#diff-fbf08688ebc9e605cd4f957a78f21ad277a1f30dc1af7d2eb0e9ed73bd6849eeR37, you include a "#" before "PWASummit22", and the lack of spaces make a little more sense if it's meant to be a hashtag. Do you want to change this to "#PWASummit22"?